schemas/README.md

```diff
@@ -2335,7 +2333,52 @@ bytes
 </td>
 <td>
 
-Raw image data
+Raw image data.
+
+The requirements for each `encoding` value are:
+- `yuv422`,  `uyvy`:
+  - 8-bit [Y'UV](https://en.wikipedia.org/wiki/Y%E2%80%B2UV).
+  - U and V values are shared between horizontal pairs of pixels. Each pair of output pixels is encoded as [U, Y1, V, Y2].
+  - `step` must be greater than or equal to `width` * 2.
```

Comment:
nit: remove all unnecessary periods (anywhere you have only a single sentence or a fragment)

[Material Design writing guidelines](https://m3.material.io/foundations/content-design/style-guide/grammar-and-punctuation#555b17c7-de48-4974-8a82-51106899dc0e)

```suggestion
  - `step` must be greater than or equal to `width` * 2
```